lib/charon/session_store/redis_store.ex

```diff
@@ -55,20 +55,22 @@ defmodule Charon.SessionStore.RedisStore do
     config = get_mod_config(config)
     now = Internal.now()
     session_key = session_key(sid, uid, type, config)
+    # redis returns error on <0 ttl
+    ttl = max(exp - now, 1)
```

Comment:
I don't think it is necessary to immediately delete a session with a negative TTL. This will not cause problems in practice, because the `exp` claim of the session's tokens will still be invalid at the correct time.